src/main/java/org/rumbledb/server/RumbleHandler.java

```diff
@@ -0,0 +1,203 @@
+                output.putItemByKey("values", values);
+            }
+            output.putItemByKey("status", ItemFactory.getInstance().createIntegerItem(200));
+            if (configuration.getOutputPath() != null) {
+                output.putItemByKey(
+                    "output-path",
+                    ItemFactory.getInstance().createStringItem(configuration.getOutputPath())
+                );
+            }
+            if (configuration.getLogPath() != null) {
+                output.putItemByKey("log-path", ItemFactory.getInstance().createStringItem(configuration.getLogPath()));
+            }
+
+            sendResponse(exchange, StatusCode.SUCCESS, output.serialize());
+        } catch (Exception e) {
+            Item output = handleException(e);
+            sendResponse(exchange, StatusCode.SUCCESS, output.serialize());
+        }
+    }
+
+
+    private static Item handleException(Throwable ex) {
+        try {
+            if (ex != null) {
+                if (ex instanceof SparkException) {
+                    Throwable sparkExceptionCause = ex.getCause();
+                    if (sparkExceptionCause != null) {
+                        return handleException(sparkExceptionCause);
+                    }
+                    return handleException(new RumbleException(ex.getMessage()));
+                } else if (ex instanceof RumbleException && !(ex instanceof OurBadException)) {
+                    Item output = ItemFactory.getInstance().createObjectItem();
+                    output.putItemByKey("error-message", ItemFactory.getInstance().createStringItem(ex.getMessage()));
+                    output.putItemByKey(
+                        "error-code",
+                        ItemFactory.getInstance().createStringItem(((RumbleException) ex).getErrorCode())
+                    );
+                    Item stackTrace = ItemFactory.getInstance().createArrayItem();
+                    output.putItemByKey("stack-trace", stackTrace);
+                    for (StackTraceElement e : ex.getStackTrace()) {
+                        stackTrace.append(ItemFactory.getInstance().createStringItem(e.toString()));
+                    }
+                    return output;
+                } else {
+                    Item output = ItemFactory.getInstance().createObjectItem();
+                    output.putItemByKey(
+                        "error-message",
+                        ItemFactory.getInstance()
+                            .createStringItem(
+                                "Unexpected error: "
+                                    + ex.getMessage()
+                                    + " We should investigate this. Please contact us or file an issue on GitHub with your query."
+                            )
+                    );
+                    output.putItemByKey("error-code", ItemFactory.getInstance().createStringItem("RBST0004"));
```

Comment:
Can we replace the magic value of 'RBST0004' with `ErrorCode.OurBadErrorCode`

Reply:
DONE